features/bootstrap/DatasetViewContext.php

```diff
@@ -217,6 +217,34 @@ public function iShouldSeeAButton($arg1)
         PHPUnit_Framework_Assert::assertTrue( $this->minkContext->getSession()->getPage()->hasLink($arg1) );
     }
 
+    /**
+     * @Then I manually hit return
+     */
+    public function iManuallyHitReturn()
```

Comment:
This function can be deleted now because it is replaced by the `iHitReturn()` function?

Reply:
`iHitReturn` is deleted now.